cmd/metrics_exporter/metrics_exporter.go

```diff
@@ -42,6 +42,8 @@ var createMetricsExporterCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create Metrics Exporter Config",
 	Long:  "Create Metrics Exporter Config",
+	Deprecated: "please use \"integration create\" instead." +
```

Comment:
```suggestion
	Deprecated: "Use \"integration create\" instead." +
```

Reply:
not capitalized because there's a prefix.
Now looks like 
```
Command "list" is deprecated, use "integration list" instead
This command was deprecated 2024-11-30 and will be supported until 2025-2-28.